core/Pos/System/Metrics/Constants.hs

```diff
@@ -0,0 +1,13 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
This extension (among [many others](https://github.com/input-output-hk/cardano-sl/blob/master/core/cardano-sl-core.cabal#L189)) is enabled by default.

Reply:
Ah, I must have been taken in by flycheck or something 😉